media/css/firefox/channel.less

```diff
@@ -184,17 +184,19 @@
 // Platform navigation
 
 .platform-nav {
+    .sticky;
```

Comment:
By the way, you'll need to add a separate `@supports` rule for both `position: sticky` and `-webkit-position: sticky`. Unfortunately our CSS minifier chokes on using `or` 😞.